lib/hanami/model/plugins/timestamps.rb

```diff
@@ -24,7 +24,7 @@ class InputWithTimestamp < WrappingInput
           def initialize(relation, input)
             super
             columns     = relation.columns.sort
```

Comment:
Yep, that's what I meant by "what's the point of [...] instantiating `columns`" ;)

Anyway, that's cleaned up too